css/style.css

```diff
@@ -18,13 +18,12 @@ body h3 {
 /* ↓HEADER↓ */
 
 header {
+  width: 100vw;
+  box-sizing: border-box;
+  padding: 1vh 24px 0;
   position: fixed;
-  width: 100%;
   background-color: #3c3a39;
   height: 5vh;
```

Comment:
- [ ] Kindly remove the `height: 5vh;` role, because it makes a problem with your design.

![image](https://user-images.githubusercontent.com/35535130/154542457-5e6f0456-c98e-456c-8cb3-50e350855429.png)

Reply:
thanks, I didn't notice that before.